File: coding-practice/50 Coding Challenges part 1/C#/problem23.cs
// Coding challenge #23 : Reverse a string
using System;
using System.Text;

class problem23
{    
    static string reverse_string(string input)
    {
        int len = input.Length;
        StringBuilder output = new StringBuilder(len);
        for (int i = 0; i < len; i++)
        {
            output.Append(input[(len - 1) - i]);
        }
        return output.ToString();
    }

    static void Main(string[] args)
    {
        string word = "Ramadhani";
        string reversed_word = reverse_string(word);
        Console.Write("The reversed version of " + word + " is : " + reversed_word);
    }
}

File: coding-practice/50 Coding Challenges part 1/Python/problem23.py
# Coding challenge 23 : Reverse a string
myString = 'Ramadhani Abidin'
newString = ''
i = len(myString)

while(i > 0):
    newString += myString[i -1]
    i -= 1

print(f'Original string: {myString}')
print(f'Reversed string: {newString}')